runtime/Python3/src/antlr4/Recognizer.py

```diff
@@ -49,14 +49,20 @@ def removeErrorListener(self, listener):
     def removeErrorListeners(self):
         self._listeners = []
 
+    def getTokenNames(self):
+        return tuple(self.literalNames + self.symbolicNames[len(self.literalNames):])
```

Comment:
same here

Reply:
Let's discuss in the other thread and I'll update here as necessary.